docker/test_dockerimage.sh

```diff
@@ -13,5 +13,15 @@ emcc --version
 java -version
 cmake --version
 
+mkdir -p test
+
+# test emcc compilation
+echo 'int main() { return 0; }' > test/main.c
+emcc -c test/main.c -o test/main
+
+# test embuilder
+embuilder build zlib
```

Comment:
Should we test this doesn't actually build anything?  (since the re-loaded cache should contain these libraries).

Reply:
I added the `--force` parameter with commit https://github.com/emscripten-core/emsdk/pull/571/commits/d7021674357cf0d3bfa352e7eafc2733c2de32a7 to ensure it's not built from the cache.